SU-HSDO/suhumsci
# SHS-4993: Revert warbler color changes

# READY FOR REVIEW

## Summary
- Revert warbler palette color changes due to an unexpected side effect in card titles

## Need Review By (Date)
- 09/11

## Urgency
- medium

## Steps to Test
1. Visit the homepage of the Anthropology site
2. Confirm that the titles of the cards in the "Recent News" are black

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
